install.sh

```diff
@@ -317,6 +317,10 @@ git fetch origin "${BRANCH}"
 git reset --hard origin/"${BRANCH}"
 crew update
 
+# install supporting packages for llvm-strip 
+# (The install for these above doesn't seem to work properly.)
+yes | crew install libffi icu4c
```

Comment:
This shouldn't be needed.  Try a fresh Docker container test again.

Reply:
Hmm. It's only a cosmetic issue. But I get a libffi not found otherwise when building a new container. Weird, right?

Reply:
Setting a rebuild to go again and hitting the sack. Aside from buildessential failing the first time it is run due to yes not being passed correctly, it works for me since I force buildessential to be installed twice.

Reply:
huh. works now.  new image.bin from recovery image and problem resolved.